docker-images/ziti-console/README.md

```diff
@@ -0,0 +1,37 @@
+
+# openziti/ziti-console
+
+## Overview
+
+This is the preferred image for including the Ziti Administration Console (ZAC) in a multi-stage build of the ziti-controller image. The controller has an optional API binding `zac` that creates a `/zac` handler on the `edge-management` API's web listener.
+
+## Deploy
+
+[Link to deployment guide](https://openziti.io/docs/guides/deployments/docker/console)
+
+## Build
+
+In the project root, build the image with the `BASE_HREF` build argument set to the path where the console will be served (default: `/zac` is hard-coded in `ziti controller`).
+
+The default value of `DEPLOY_URL` is the value of `BASE_HREF`. These values are used in the Angular build to set the base URL for the console and correspond to `ng` CLI parameters `--base-href` and `--deploy-url`, respectively.
```

Comment:
whole paragraph seems like exposition of what BASE_HREF does. more words that aren't adding particular value over the first paragraph and can just be removed. i'd prefer linking to some angular document where people can 'learn more' if they want to instead of trying to explain it to them